pwndbg/dbg/__init__.py

```diff
@@ -13,6 +13,18 @@
 dbg: Debugger = None
 
 
+class Registers:
+    """
+    A handle to the register values in a frame.
+    """
+
+    def by_name(self, name: str) -> Value | None:
```

Comment:
Honestly, I don't like having classes with one methods which are unimpl. Is there much benefit of having a base class instead of having function per each debugger for all of this?

I mean, there is some mantra/saying that "class with just one method should be a function" and I kinda agree with it :P.

Reply:
Still why do we need a class at all for that?

Wouldn't it be better to just call GetRegisters on each stop and call it a day?